README.md

```diff
@@ -123,8 +123,54 @@ public static void StreamsTrigger(
 }
 ```
 
-## Known Issues
-- The `RedisPubSubTrigger` is not capable of listening to [keyspace notifications](https://redis.io/docs/manual/keyspace-notifications/) on clustered caches.
+### `Redis` Input Binding
+
+#### Inputs
+- `ConnectionStringSetting`: Name of the setting in the appsettings that holds the to the Redis cache connection string (eg `<cacheName>.redis.cache.windows.net:6380,password=...`).
+- `Command`: The redis-cli command to be executed on the cache with all arguments separated by spaces.
+
+> **Note**
+> Not all commands are supported for this binding. At the moment, only read commands that return a single output are supported.
+
+#### Avaiable Output Types
+- [`StackExchange.Redis.RedisValue`](https://github.com/StackExchange/StackExchange.Redis/blob/main/src/StackExchange.Redis/RedisValue.cs), `string`, `byte[]`, `ReadOnlyMemory<byte>`: The value returned by the command.
+- `Custom`: The trigger uses Json.NET serialization to map the value returned by the command from a `string` into a custom type.
+
+#### Sample
+The following gets any key that was recently set.
+```c#
+[FunctionName(nameof(SetGetter))]
+public static void SetGetter(
+    [RedisPubSubTrigger("Redis", "__keyevent@0__:set")] string key,
+    [Redis("Redis", "GET {Message}")] string value,
```

Comment:
How do we handle parameters that can be passed in with commands? Is everything stuffed into the command, or are they passed in through other means?

Reply:
For multiple parameters, it would be `HGET key field` with spaces in between each value